### PR TITLE
feat: [SRM-13091]: Update preview for Add SLO and Target

### PR DESCRIPTION
**Pull request by deepesh-ui**
### Summary

Preview for Composite SLO create section 

#### Screenshots

Screenshot 2022-11-17 at 6 03 53 PM

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/pages/slos/components/CVCreateSLOV2/components/CreateCompositeSloForm/components/CreatePreview/CreatePreview.tsx [hunk 1]
```diff
@@ -64,31 +81,42 @@ export const CreatePreview = ({ id, data }: { id: CreateCompositeSLOSteps; data:
...
-            return (
-              <LabelAndValue
-                key={slo.serviceLevelObjectiveRef}
-                label={slo.serviceLevelObjectiveRef}
-                value={slo.weightagePercentage.toString()}
-              />
-            )
-          })}
-        </Layout.Vertical>
+        <>
```
Comment (praneshg239) [inline src/modules/85-cv/pages/slos/components/CVCreateSLOV2/components/CreateCompositeSloForm/components/CreatePreview/CreatePreview.tsx:98]: remove fragment

File: src/modules/85-cv/components/CVStepper/components/Step/Step.tsx [hunk 2]
```diff
@@ -66,7 +66,7 @@ const Step = ({
 
   return (
     <>
```
Comment (praneshg239) [inline src/modules/85-cv/components/CVStepper/components/Step/Step.tsx:68]: remove fragment

File: src/modules/85-cv/pages/slos/components/CVCreateSLOV2/components/CreateCompositeSloForm/components/CreatePreview/CreatePreview.tsx [hunk 3]
```diff
@@ -9,33 +9,50 @@ import React from 'react'
...
+import css from './CreatePreview.module.scss'
 
-export const LabelAndValue = ({ label, value }: { label: string; value: string }): JSX.Element => {
+export const LabelAndValue = ({
+  label,
+  value,
+  className,
+  isLabelHeading = true,
+  isValueHeading = false
+}: {
```
Comment (praneshg239) [inline src/modules/85-cv/pages/slos/components/CVCreateSLOV2/components/CreateCompositeSloForm/components/CreatePreview/CreatePreview.tsx:24]: move type to types file

**Review by praneshg239 — CHANGES_REQUESTED** [review]
please check the comments

**Review by praneshg239 — APPROVED** [review]

**Review by karansaraswat19 — APPROVED** [review]

**Pull request merged**